src/pages/index.js

```diff
@@ -85,98 +61,137 @@ const previewModalCloseBtn = previewModal.querySelector(".modal__close-btn");
 const cardTemplate = document.querySelector("#card-template");
 const cardsList = document.querySelector(".cards__list");
 
-// Avatar Modal Setup
 const avatarModal = document.querySelector("#avatar-modal");
 const avatarForm = avatarModal.querySelector("#edit-avatar-form");
 const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
 const avatarModalBtn = document.querySelector(".profile__avatar-btn");
 const avatarInput = avatarModal.querySelector("#profile-avatar-input");
 
-// Delete form elements
 const deleteModal = document.querySelector("#delete-modal");
 
-// Add this similar to other modals
 avatarModal.addEventListener("click", (event) => {
   if (event.target.classList.contains("modal")) closeModal(avatarModal);
 });
 
-// When opening avatar modal, reset validation
 avatarModalBtn.addEventListener("click", () => {
   resetValidation(avatarForm, settings);
   const submitButton = avatarForm.querySelector(".modal__submit-btn");
   disabledButton(submitButton, settings);
   openModal(avatarModal);
 });
 
-// Close the avatar modal when the close button is clicked
 avatarModalCloseBtn.addEventListener("click", () => {
   closeModal(avatarModal);
 });
 
-// Add submit event listener
 avatarForm.addEventListener("submit", (evt) => {
-  console.log("Form submit triggered"); // Debugging log
+  evt.preventDefault();
   handleAvatarSubmit(evt);
 });
 
 function handleAvatarSubmit(evt) {
```

Comment:
* According to the task, you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.